docs/index.html

```diff
@@ -13,9 +13,9 @@
     <link type="text/css" rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
     <link type="text/css" rel="stylesheet" href="styles/prettify-tomorrow.css">
     <link type="text/css" rel="stylesheet" href="styles/jsdoc-default.css">
-    <link rel="icon" 
-      type="image/png" 
-      href="img/antispam.png">
+	<link rel="icon" 
```

Comment:
can you make sure the indentation is consistent here? and in general clean up this file